rbc/targetinfo.py

```diff
@@ -398,7 +398,14 @@ def sizeof(self, t):
             return
         if isinstance(t, str):
             if t == 'int':
+                warnings.warn('Using sizeof(int) == 4')
                 return 4  # this is a guess
+            if t == 'long':
+                warnings.warn('Using sizeof(long) == 4')
+                return 4  # this is a guess
```

Comment:
I suggest using 8:
```python
>>> import ctypes
>>> ctypes.sizeof(ctypes.c_long)
8
```
```

Reply:
I used the following table to derive those numbers: https://en.cppreference.com/w/cpp/language/types#Integer_types